index.js

```diff
@@ -835,6 +871,68 @@ class Batch {
   }
 }
 
+class Watcher extends EventEmitter {
+  constructor (bee, range) {
+    super()
+
+    this.bee = bee
+    this.core = bee.core
+
+    this.running = false
+    this.destroyed = false
+
+    this.range = range
+    this.latestDiff = this.bee.version
+    this.stream = null
+
+    this._onappend = this._onappend.bind(this)
+    this._onappend = debounceify(this._onappend)
+
+    // this.on('error', noop)
```

Comment:
Leaving this to @mafintosh to decide if it should ignore errors by default, otherwise watcher can throw errors due `emit('error', err)` but only unexpected ones